frontend/src/helpers/connector.js

```diff
@@ -5,12 +5,11 @@ import { useState, useEffect } from 'react'
 export const MetaMask = new InjectedConnector({ supportedNetworks: [1, 3, 4, 5, 42, 421611] })
 
 const contracts = {
-  1337: {"vault": "0x724Ca58E1e6e64BFB1E15d7Eec0fe1E5f581c7bD", "eth": "0xE7eD6747FaC5360f88a2EFC03E00d25789F69291", "dai": "0x6951b5Bd815043E3F842c1b026b0Fa888Cc2DD85"}, //mainnet-fork
-  3: {"vault": "0x130c973Bbe11CBc5BAE094a45710CDE4Bebb8438", "eth": "0xc778417e063141139fce010982780140aa0cd5ab", "dai": "0xaD6D458402F60fD3Bd25163575031ACDce07538D"}, // Ropsten
+  1337: {"vault": "0xe65A7a341978d59d40d30FC23F5014FACB4f575A", "eth": "0x7231BBeCcAd62eB462f2A126f562fd4af08A5e06", "dai": "0xb8d4205617c43EfA32481D23eA91A5f9B3B7B2b1"}, //mainnet-fork
+  3: {"vault": "0x130c973Bbe11CBc5BAE094a45710CDE4Bebb8438", "eth": "0x7F88e71C8aE4EB91AE678D2dd5dE6d7Bd7d5A019", "dai": "0x6c0f5CEf5dCF88F591C97f2BA9ba14B54727aae5"}, // Ropsten vault: 0x130c973Bbe11CBc5BAE094a45710CDE4Bebb8438
```

Comment:
you are chaning Ropsten vault address? why?

Reply:
I was testing with ropsten too and redeployed on ropsten because i needed to mint myself some weth and dai